### PR TITLE
JCS Algorithms file

### PR DESCRIPTION
**Pull request by PatStLouis**
Adds a couple helper functions and coverage for jcs statements.

### CODE REVIEW

**Review by TallTed — CHANGES_REQUESTED** [review]

File: config/runner.json [hunk 1]
```diff
@@ -8,7 +8,17 @@
...
       },
       "interop": {
-        "vcVersion": "1.1"
+        "vcVersion": "2.0"
+      }
+    },
+    "ecdsa-jcs-2019": {
+      "tags": ["ecdsa-jcs-2019"],
+      "proofLengths": {
+        "P-256": 64,
```
Comment (aljones15) [inline config/runner.json:17]: double check these proofLengths, but they probably work as is.

**Comment by aljones15** [comment]
the tests runs and don't conflict with the other tests so I am marking this ready for review.
It needs to take multiple credentials, but working tests are working tests.

File: tests/helpers.js [hunk 2]
```diff
@@ -90,7 +92,7 @@ export const endpointCheck = ({endpoint, vcVersion, keyType}) => {
   const {
     supportedEcdsaKeyTypes,
     // assume support for vc 1.1
```
Comment (BigBlueHat) [inline tests/helpers.js:94]:
```suggestion
    // assume support for vc 2.0
```

**Review by BigBlueHat — APPROVED** [review]
Great work, as always!

**Pull request merged**